### ISSUE TITLE
Add a command to list all members that have a specific role

### ISSUE THREAD

**Issue by TSBots** [r-Techsupport/TechSupportBot#9]

**Comment by Cpt-Dingus** [comment]
Are we sure we still want to implement this? The scale at which we operate would need tens of thousands of users be checked, that would cause a ton of strain on the bot

**Comment by Cpt-Dingus** [comment]
Yeah on second thought definitely no, the [api reference for guilds](https://discordpy.readthedocs.io/en/stable/api.html#discord.Guild.members) states that the returned parameter when getting members is a list, which would need the bot iterating over 35.000+ entries.

**Comment by Cpt-Dingus** [comment]
Actually I'll make this, but I'll note the fact that it's taxing and can take a bit
